### DISCUSSION TITLE
Short URLs

### DISCUSSION THREAD

**Discussion by trymeouteh** [PrivateBin/PrivateBin#929] [Ideas]
A option in pastes to generate a short URL instead of a long URL. And by choosing a short URL, maybe decreasing the expiry to a week at max to prevent the possibility of running out of short URLs and the ability to reuse short URLs.

Also by choosing a short URL, it could require the user to enter a password which is used to encrypt the paste instead of a long string for the password.

**Comment by rugk** [comment]
You can use an external URL shorter for that. Please see [this FAQ question](https://github.com/PrivateBin/PrivateBin/wiki/FAQ#the-url-is-so-long-cant-i-just-use-an-url-shortener) for details and how/what is possible. 
Password and expiration time can be adjusted by the user using this service then, it's a security trade-off as described in the FAQ and not very much recommend. Indeed, it is recommend to set/use a password at least then…

Actually, the best idea then is to use a "destroy after reading" paste.
